### DISCUSSION TITLE
How to "unsupport"  TLS 1.0 and TLS 1.1.

### DISCUSSION THREAD

**Discussion by wawefr** [NginxProxyManager/nginx-proxy-manager#2944] [Q&A] [answered]
Hello guy's
When I made a test with SSL Lab I have a subject about TLS 1.0 and 1.1.
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/62282235/73ca8499-93a9-4d3a-855c-e41b171ad711)

I change the ssl-ciphers.conf with this config but always the problem

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/62282235/0df61a94-d454-4767-9733-5d6dcc293b65)

If you can help me to stop TLS1.0 and TLS1.1
Thanks Fred

**Comment by rafabravo** [comment, marked as answer]
Hi,
I have added in "advanced":

ssl_protocols TLSv1.2 TLSv1.3;

and... 

Protocols
TLS 1.3	No
TLS 1.2	Yes
TLS 1.1	No
TLS 1.0	No
SSL 3	No
SSL 2	No

But it doesn't enabled tls1.3...

**Reply by wawefr** [reply]
Hi thanks. Where dose I write "advanced" ? wich file ? Thanks

**Reply by rafabravo** [reply]
Yes, in advanced:

**Reply by rafabravo** [reply]
My config:

ssl_protocols TLSv1.2 TLSv1.3;
ssl_dhparam /data/nginx/dhparam.pem;  <== create 
ssl_ciphers TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES256-GCM-SHA384:!TLS_AES_128_GCM_SHA256;
ssl_prefer_server_ciphers off;
ssl_ecdh_curve secp521r1:secp384r1:prime256v1;
ssl_session_timeout  10m;
ssl_session_cache shared:SSL:10m;
resolver_timeout 5s;
add_header X-Frame-Options DENY;
add_header X-Content-Type-Options nosniff;
add_header X-XSS-Protection "1; mode=block";

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/26482283/2e103ee6-d927-4051-94fc-407026ac49cd)

**Comment by rafabravo** [comment]
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/2823

**Comment by wawefr** [comment]
Hi 
Many thanks for your answer I will do a test 
At this moment I finded an other solution via Cloudflare tunnel where I can block TLS TLS 1.1 and TLS 1.0 
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/62282235/7dbab589-e1d9-4912-9c77-19848fa5e540)
